src/reactviews/pages/ExecutionPlan/executionPlanGraph.tsx

```diff
@@ -0,0 +1,203 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useEffect, useState } from 'react';
+import { ExecutionPlanContext } from "./executionPlanStateProvider";
+import * as utils from './queryPlanSetup';
+import * as azdataGraph from 'azdataGraph/dist/build';

```

Comment:
do we need to reference `dist/build` to import this control?  Can't we do something like `import * as azdataGraphModule from 'azdataGraph';` like we do in ADS?

Reply:
The import statement and require() didn't work in this context. So this was the next best thing we came up with

Reply:
Basically, when we tried to import it normally, typescript would not be able to find a type definition for it. When we added one in, the package would not load correctly, and was not usable. the require didn't work for similar reasons. You can probably ask @aasimkhan30  for more context

Reply:
Do you know why the import syntax I showed working ADS?

Reply:
@kburtram , that's because we add specify that alias in the loader. 
https://github.com/microsoft/azuredatastudio/blob/db96392893698710b4a4280a938022b2b4807894/src/vs/code/browser/workbench/workbench.html#L119

We can probably add something similar in the esbuild options but I agree that the right fix will be in the control itself.

Reply:
ok, makes sense.  Maybe this is something we can clean-up later in the azgraph module.